api/user.go

```diff
@@ -16,19 +16,21 @@ func (r GetUserRequest) ValidationRules() []validate.ValidationRule {
 }
 
 type User struct {
-	ID            uid.ID   `json:"id" note:"User ID"`
-	Created       Time     `json:"created"`
-	Updated       Time     `json:"updated"`
-	LastSeenAt    Time     `json:"lastSeenAt"`
-	Name          string   `json:"name" note:"Name of the user" example:"bob@example.com"`
-	ProviderNames []string `json:"providerNames,omitempty" note:"List of providers this user belongs to" example:"['okta']"`
+	ID            uid.ID          `json:"id" note:"User ID"`
+	Created       Time            `json:"created"`
+	Updated       Time            `json:"updated"`
+	LastSeenAt    Time            `json:"lastSeenAt"`
+	Name          string          `json:"name" note:"Name of the user" example:"bob@example.com"`
+	ProviderNames []string        `json:"providerNames,omitempty" note:"List of providers this user belongs to" example:"['okta']"`
+	PublicKeys    []UserPublicKey `json:"publicKeys,omitempty" note:"List of the users public keys"`
 }
 
 type ListUsersRequest struct {
-	Name       string   `form:"name" note:"Name of the user" example:"bob@example.com"`
-	Group      uid.ID   `form:"group" note:"Group the user belongs to" example:"admins"`
-	IDs        []uid.ID `form:"ids" note:"List of User IDs"`
-	ShowSystem bool     `form:"showSystem" note:"if true, this shows the connector and other internal users" example:"false"`
+	Name                 string   `form:"name" note:"Name of the user" example:"bob@example.com"`
+	Group                uid.ID   `form:"group" note:"Group the user belongs to" example:"admins"`
+	IDs                  []uid.ID `form:"ids" note:"List of User IDs"`
+	ShowSystem           bool     `form:"showSystem" note:"if true, this shows the connector and other internal users" example:"false"`
+	PublicKeyFingerprint string   `form:"publicKeyFingerprint" note:"Find the user with a public key that matches this SHA256 fingerprint."`
```

Comment:
I don't think we have a use case for searching for multiple public keys. A user may have multiple groups as well, but `Group` here is also a single value, because generally we only query by 1 value at a time.  This is the `ListUsersRequest` struct